README.md

```diff
@@ -11,8 +12,10 @@ For algorithms that use a conic solver instead of an NLP solver, use [Pajarito](
 ## Installation
 
 Pavito can be installed through the Julia package manager:
+
 ```
-julia> Pkg.add("Pavito")
+julia> ]
+pkg> Pkg.add("Pavito")
```

Comment:
What's the point of entering th `Pkg` environment if you don't write `add Pavito` ?

Reply:
correct yes, fixed in last commit